package.json

```diff
@@ -62,6 +62,7 @@
     "lint-staged": "^10.5.3",
     "node-mocks-http": "^1.10.0",
     "prettier": "^2.2.1",
+    "prop-types": "^15.7.2",
```

Comment:
I wonder if this needs to be in `dependencies`, see https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package